integration-test/nodejs-esbuild/serverless.yml

```diff
@@ -0,0 +1,21 @@
+service: serverless-plugin-nodejs-layers
+
+custom:
+  lumigo:
+    token: 1234
+    useLayers: true
```

Comment:
I think that it's suppose to work without `useLayer`, we suppose to identify `serverless-esbuild` plugin.